### DISCUSSION TITLE
opencrx forum

### DISCUSSION THREAD

**Discussion by nyemiks** [opencrx/opencrx#78] [Q&A]
I have noticed that the opencrx forum on sourceforge is no longer available.
Is there a new forum on github ? If there is not I would really appreciate if one is created. I have a question to ask.
The previous forum at sourceforge was really helpful to me.

**Comment by javac9** [comment]
Hi @nyemiks , have you already seen this: https://github.com/opencrx/opencrx-documentation/discussions/15

**Comment by javac9** [comment]
@wfro64 Could you please move issues labeled as `question` to Discussions as well, as you did with the opencrx-documentation repo? Thanks!

**Comment by wfro64** [comment]
This is currently not possible because of https://github.com/github/feedback/discussions/2952
